docs/advanced/arrow_function.md

```diff
@@ -0,0 +1,113 @@
+## Arrow Functions
+Arrow functions are functions defined using the arrow (=>) symbol, and they are commonly used to create anonymous functions or to pass as callback functions to other functions.
+### Supported Patterns
+
+```yaml
+name: Advanced Arrow Functions
+```
+
+#### Semantic: Arrow Functions
+
+##### Examples
+
+###### Call + Chained call
+ lambda functions in Python are represented using arrow functions in JavaScript.
```

Comment:
In the descriptive paragraph, there is no need to mention Python. Please remove all sentences relevant to Python.